packages/wordpress-theme-boilerplate/lib/commands/create.js

```diff
@@ -0,0 +1,280 @@
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'themeName',
+			default: 'My Theme',
+			message: 'Enter the name of the theme:',
+			validate: validateNotEmpty,
+		},
+		{
+			type: 'input',
+			name: 'themeDescription',
+			default: '',
+			message: 'Enter the description of the theme:',
+		},
+		{
+			type: 'input',
+			name: 'themeSlug',
+			default: ( answers ) => paramCase( answers.themeName ),
+			message: 'Enter the slug of the theme:',
+			validate: validateSlug,
+		},
+		{
+			type: 'input',
+			name: 'phpNamespace',
+			default: ( answers ) => 'Required\\' + pascalCase( answers.projectSlug ),
+			message: 'Enter the PHP namespace of the theme:',
+			validate: validatePHPNamespace,
+		},
+		{
+			type: 'input',
+			name: 'githubSlug',
+			default: ( answers ) => answers.themeSlug,
+			message: 'Enter the slug of the GitHub repo:',
+			validate: validateSlug,
+		},
+		{
+			type: 'confirm',
+			name: 'privateRepo',
+			default: true,
+			message: 'Private GitHub repo?',
+		},
+	] );
+
+	log();
+
+	const themeDir = WORKING_DIR + '/' + themeSlug;
+
+	// Check if theme slug doesn't already exist in working directory.
+	let themeDirExists;
+	try {
+		await fs.stat( themeDir );
+		themeDirExists = true;
+	} catch ( e ) {
+		themeDirExists = false;
+	}
+
+	if ( themeDirExists ) {
+		log( format.error( themeDir + ' already exists, please delete first.' ) );
+		process.exit();
+	}
+
+	// Add token to the keychain.
+	if ( storedGithubToken !== githubToken ) {
+		await keytar.setPassword( 'wordpress-theme-boilerplate', 'github', githubToken );
+	}
+
+	const octokit = new Octokit( {
+		auth: githubToken,
+		previews: [ 'baptiste' ],
+	} );
+
+	// Create the repository.
+	let response;
+	await runStep( 'Creating repository using template', 'Could not create repo.', async () => {
+		response = await octokit.repos.createUsingTemplate( {
+			template_owner: TEMPLATE_OWNER,
+			template_repo: TEMPLATE_REPO,
+			owner: TEMPLATE_OWNER,
+			name: githubSlug,
+			private: privateRepo,
+			description: themeDescription,
+		} );
+	} );
+
+	const { data: repo } = response;
+
+	// Update repo to disbale issues, projects and wiki.
+	await runStep( 'Disabling issues, projects & wikis in repository', 'Could not update repo.', async () => {
+		await octokit.repos.update( {
+			owner: repo.owner.login,
+			repo: repo.name,
+			has_issues: false,
+			has_projects: false,
+			has_wiki: false,
+		});
+	} );
+
+	const git = simpleGit();
+
+	// Clone the repository.
+	await runStep( 'Cloning repository into a new directory', 'Git checkout failed.', async () => {
+		// Give GitHub some time to create the repository
+		// to prevent cloning an empty repository.
+		await sleep( 3000 );
+		await git.clone( repo.ssh_url, themeDir );
+	} );
+
+	// Rename files in local checkout.
+	await runStep( 'Renaming theme files', 'Could not rename files.', async () => {
+		// await fs.writeFile( themeDir + '/README.md', '# ' + ThemeName + '\n' ); // @TODO
```

Comment:
What's missing here?

Reply:
The [plugin boilerplate README](https://github.com/wearerequired/wordpress-plugin-boilerplate/blob/master/README.md) is about setting up the plugin and not the about theme.

I was not sure if I should do the same for the Theme but decided against it. I will just have the [Theme README](https://github.com/wearerequired/wordpress-theme-boilerplate/blob/main/README.md) about the Theme and not the setup. The documentation of the setup can stay in the JS monorepo.